### PR TITLE
chore(samples): Retail Tutorials. Import products

### PR DESCRIPTION
**Pull request by sborisenkox**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

**Review by Neenu1995 — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/product/ImportProductsBigQueryTable.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            .setDataSchema(DATA_SCHEMA)
+            .build();
+
+    ProductInputConfig inputConfig =
+        ProductInputConfig.newBuilder().setBigQuerySource(bigQuerySource).build();
+
+    ImportProductsRequest importRequest =
+        ImportProductsRequest.newBuilder()
+            .setParent(DEFAULT_CATALOG)
+            .setReconciliationModeValue(reconciliationMode.getNumber())
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/ImportProductsBigQueryTable.java:109]: Can we just setReconciliationMode(reconciliationMode)?

File: samples/interactive-tutorials/src/main/java/product/ImportProductsInlineSource.java [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+    FieldMask fieldMask2 =
+        FieldMask.newBuilder()
+            .addAllPaths(Arrays.asList("title", "categories", "price_info", "color_info"))
+            .build();
+
+    product2 =
+        Product.newBuilder()
+            .setTitle("Android Embroidered Crewneck Sweater")
+            .setId(GENERATED_PRODUCT_ID)
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/ImportProductsInlineSource.java:199]: Should product 2 use a different ID than product 1?

**Review by chenlei1216 — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/product/ImportProductsBigQueryTable.java (hunk 1 above)
Comment (pmichalskiGD) [reply samples/interactive-tutorials/src/main/java/product/ImportProductsBigQueryTable.java]: fixed

File: samples/interactive-tutorials/src/main/java/product/ImportProductsInlineSource.java (hunk 2 above)
Comment (pmichalskiGD) [reply samples/interactive-tutorials/src/main/java/product/ImportProductsInlineSource.java]: yes, you're right. fixed

**Pull request merged**